src/library-authoring/author-library/LibraryAuthoringPage.jsx

```diff
@@ -324,6 +330,75 @@ const deriveTypeOptions = (blockTypes, intl) => {
   return typeOptions;
 };
 
+
+/**
+ * LibraryAuthoringPageHeaderBase
+ * Title component for the LibraryAuthoringPageBase.
+ */
+const LibraryAuthoringPageHeaderBase = ({intl, library, ...props}) => {
+  const [inputIsActive, setIsActive] = useState(false);
+  const handleOnBlur = (event) => {
+    var newTitle = event.target.value;
+    if (newTitle && newTitle != library.title){
+      props.updateLibrary({ data: { title: newTitle, libraryId: library.id } });
+    }
+    setIsActive(false);
+  };
+  const handleClick = (event) => {
+    setIsActive(true);
+  };
+
+  return (
+    <>
+      <small className="card-subtitle">{intl.formatMessage(messages['library.detail.page.heading'])}</small>
+      <h1 className="page-header-title">
+        {inputIsActive
+          ? (
+            <>
```

Comment:
This brackets is redundant, remove please

Reply:
Updated